account_credit_control/models/credit_control_run.py

```diff
@@ -86,6 +90,9 @@ def _compute_credit_control_count(self):
                   for data in fetch_data}
         for rec in self:
             rec.credit_control_count = result.get(rec.id, 0)
+            rec.credit_control_communication_count = len(
```

Comment:
Please don't add this non optimized way. Can you do a `read_group`?

Reply:
Not here, because I'd need to count distinct values for the `communication_id` field. The only possibly optimized way to get that AFAIK is raw SQL. Since this is only to be executed in form views, which shouldn't have many values, I didn't see this as a big problem.

Should I do it through SQL?

Reply:
Well, you could add the one2many to communication and add the credit_control_run_id on communication